packages/json-schema/samples/basic/ajv.js

```diff
@@ -0,0 +1,72 @@
+import Ajv from "ajv/dist/2020.js";
+
+const ajv = new Ajv({
+  schemas: [
+    {
+      $id: "hi",
+      type: "array",
+      contains: {
+        type: ["string", "number"],
+      },
+      minContains: 2,
+      maxContains: 3,
+    },
+  ],
+});
+
+const validate = ajv.getSchema("hi");
+console.log(validate([1, 1]));
+
+/*
```

Comment:
Commented code

Reply:
Gonna delete this whole file, it's really just for testing purposes.